src/vanilla/Model/CodeModelTS.cs

```diff
@@ -30,6 +30,15 @@ public class CodeModelTS : CodeModel
         [JsonIgnore]
         public string ContextName => Name + "Context";
 
+        [JsonIgnore]
+        public string ApiVersionPackageName => Settings.PackageName + "-" + Settings.ApiVersion;
```

Comment:
Can't you remove this and just use the next property (`PackageName`)?

Reply:
No